### DISCUSSION TITLE
Where to see all the supported models from Hugging Face?

### DISCUSSION THREAD

**Discussion by ilijaplusx** [explosion/spaCy#6894] [Help: Coding & Implementations] [answered]
I'm trying to find a list of pretrained models supported by `spacy-transformers`. In the repo readme it says that Hugging Face transformers are used, but I don't see which of those are supported, and how to refer to them? Per [this list of models](https://spacy.io/models), there are a couple of single-language transformers for popular languages. I'd be interested in a multilingual one though. Is there a way to refer to a Hugging Face pretrained model, eg. `bert-base-multilingual-cased`, or you need to create a wrapper around each pretrained transformer first?

**Comment by ines** [comment, marked as answer]
> Is there a way to refer to a Hugging Face pretrained model, eg. bert-base-multilingual-cased, or you need to create a wrapper around each pretrained transformer first?

In spaCy v3, `spacy-transformers` lets you include any transformer weights that can be loaded via the Transformers `AutoModel` and  `AutoTokenizer` helpers. In your [training config](https://spacy.io/usage/training#quickstart), you can provide the name of the transformer to use via the `name` setting:

```ini
[components.transformer.model]
@architectures = "spacy-transformers.TransformerModel.v1"
name = "roberta-base"
tokenizer_config = {"use_fast": true}
```

(In spaCy v2, we prepackaged some transformers, but none of that is necessary anymore. You can now just load in whatever you want to use during training. Also see #6790.)

**Reply by ilijaplusx** [reply]
Thanks @ines for your answer! I'm kinda new to spaCy, and initially confused pretrained pipelines listed [here](https://spacy.io/models), to pretrained transformer models from Hugging Face. Your answer and the docs you referred me to cleared things for me.

(I cannot seem to be able to mark it as "answered")

**Reply by ines** [reply]
Glad it worked! (I think it can't be answered because the topic is in an "open-ended" category – but I'll move it to the v3 category because it fits much better in there, and it also has a clear answer 🙂)
